newIDE/app/src/Utils/Analytics/EventSender.js

```diff
@@ -289,16 +289,30 @@ export const sendOnboardingManuallyOpened = () => {
   recordEvent('onboarding_manually_opened');
 };
 
-export const sendAssetPackOpened = (assetPackTag: string) => {
-  recordEvent('asset_pack_opened', {
-    assetPackTag,
-  });
+export const sendAssetPackOpened = (options: {|
+  assetPackName: string,
+  assetPackTag?: string,
```

Comment:
to be consistent with other functions, change type to `assetPackTag: string | null` ? (same for id)

Reply:
Done